tests/runtest.sh

```diff
@@ -58,6 +58,7 @@ function print_usage {
     echo '  --gcsimulator                    : Runs the GCSimulator tests'
     echo '  --show-time                      : Print execution sequence and running time for each test'
     echo '  --no-lf-conversion               : Do not execute LF conversion before running test script'
+    echo '  --build-overlay-only             : Build overlay directory only, do not run test'
```

Comment:
I'd change the description to something like "Exit after overlay directory is populated".